### PR TITLE
Sync 1.18 changes

### PR DESCRIPTION
**Pull request by dhaiducek**
Closes https://github.com/stolostron/governance-policy-propagator/issues/218

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -221,7 +221,7 @@ kind-bootstrap-cluster-dev: kind-create-cluster install-crds install-resources
 .PHONY: kind-deploy-controller
 kind-deploy-controller: manifests
 	@echo installing $(IMG)
-	kubectl create ns $(KIND_NAMESPACE) --dry-run=client -o yaml | kubectl apply -f -
+	kubectl create ns $(KIND_NAMESPACE)
```
Comment (JustinKuli) [inline Makefile:224]: I think these will need a `-` or they will fail if the ns already exists. Again, something we need to do upstream...

Comment (dhaiducek) [reply Makefile:224]: Yes--this will get them in sync and then we can update upstream with the behaviors we want/need.

Comment (dhaiducek) [reply Makefile:224]: Except that failing test is apparently going to force my hand here. 😆

Comment (JustinKuli) [reply Makefile:224]: Yeah 😓 I think if you put the change you want here, when it cherry-picks the same from upstream later, it should just skip the line and apply cleanly

**Review by JustinKuli — APPROVED** [review]

**Pull request merged**